mobly/base_suite.py

```diff
@@ -95,3 +95,38 @@ def setup_suite(self, config):
   def teardown_suite(self):
     """Function used to add post tests cleanup tasks (optional)."""
     pass
+
+  # Optional interfaces that users can override to record customized suite
+  # information to test summary.
+
+  def get_suite_name(self):
+    """Override to return a customized suite name (optional).
+
+    Use suite class name by default.
+
+    Returns:
+      A string that indicates the suite name.
+    """
+    return self.__class__.__name__
+
+  def get_run_identifier(self):
```

Comment:
what is "run identifier"?

Reply:
This is for key run context that will be displayed as part of the title in our result viewer.

Through this we can include info like the phone brand and phone model.